mangrove-core/how-to-guides/DirectHowTo.md

```diff
@@ -0,0 +1,307 @@
+# How to create a Direct contract
+
+This section will go through a few different ways of how a Direct contract could be implemented. If you don't know what a Direct contract is, we recommend reading both [MangroveOffer](../mangrover-offer.md) and [Direct](../Direct.md) before continuing.
+
+## Simple Direct implementation
+
+When creating a Direct contract, the first thing you need is to import the necessary contracts needed for the constructor. Next is creating a constructor that calls the constructor of Direct. We set the gas requirement to 30.000, since we are not doing much in the posthook<!-- FIXME: link to explanation of gas requirement and posthook -->, it does not need anymore than that.
+
+In the constructor we check if the deployer of the contract is the same as `msg.sender`, if not, it means that the sender is deploying the contract on behalf of another address and wants that address to be admin of the contract.
+
+```solidity
+pragma solidity ^0.8.10;
+
+pragma abicoder v2;
+
+import "mgv_src/strategies/offer_maker/abstract/Direct.sol";
```

Comment:
since remappings are local to us, maybe we want to use `src` here

Reply:
Also maybe we want to use explicit import like:
```solidity
import {Direct, AbstractRouter, IMangrove, IERC20} from "src/strategies/offer_maker/abstract/Direct.sol";
import {IMakerLogic} from "src/strategies/interfaces/IMakerLogic.sol";
```

Reply:
Done. Changed alle examples to use explicit imports and not use our remapping